test/scripts/integration_tests.py

```diff
@@ -71,6 +73,39 @@ def delete(self, filepath, raise_error=False):
         self.__run_command("delete", filepath, raise_error)
 
 
+class Helm(object):
+    """
+    Class used to interact with helm and return data
+    """
+
+    def delete(self, release_name):
+        """
+        Delete a named helm release
+        """
+        try:
+            subprocess.check_call("helm delete --purge {}".format(release_name), shell=True)
```

Comment:
nitpick: The commands above were done via a list format as opposed to single string. Both ways are valid, but probably want to pick one for consistency. I don't really have a preference, and just picked list with my initial implementation since I've done that in the past.